### DISCUSSION TITLE
Oauth: Support disabling user registrations

### DISCUSSION THREAD

**Discussion by hufman** [Kovah/LinkAce#872] [Ideas & Feature Requests]
I am enjoying LinkAce's recent Oauth support for convenient authentication to my personal LinkAce install. Thank you very much for this feature!
Can there be a config flag to disable user creation in this feature? I appreciate the usefulness of creating these remote users automatically for initial setup, but I'd like to lock the server down and close registrations after my personal account (or small group of friends) have been set up.

**Comment by hufman** [comment]
I could work on a PR [around this part of code](https://github.com/Kovah/LinkAce/blob/5db5306768781eb63655c2714609a77b4ecd95c2/app/Http/Controllers/SocialiteController.php#L42) but PHP isn't my primary language, if you'd like me to contribute a start!

**Comment by Kovah** [comment]
This is now implemented and will ship with LinkAce 2.0. Adding this to the .env file will disable registrations:
```
SSO_REGISTRATION_ENABLED=false
```

**Reply by hufman** [reply]
Oh cool, thank you so much!
